src/module/system/damage/weapon.ts

```diff
@@ -567,7 +567,7 @@ class WeaponDamagePF2e {
 
     /** Determine whether the damage source is a strength-based statistic */
     static strengthBasedDamage(weapon: WeaponPF2e | MeleePF2e): boolean {
-        return weapon.isMelee || (weapon.isThrown && !weapon.traits.has("splash"));
+        return weapon.isMelee || (weapon.isThrown && !(weapon.traits.has("splash") || weapon.traits.has("bomb")));
```

Comment:
It's specifically the splash trait that calls for withholding a character's strength ability modifier.

Reply:
Hm. Yes, you are correct. This is probably an oversight, but according to RAW it would include strength.